apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/Components/BuildDisplayItem.tsx

```diff
@@ -284,11 +289,12 @@ function CompareArtifactModal({
           <Box minWidth={320} display="flex" flexDirection="column" gap={1}>
             <ArtifactCard
               artifactId={newId}
+              onDelete={deleteArtifact}
               mainStatAssumptionLevel={mainStatAssumptionLevel}
               canEquip
               editorProps={{ disableSet: true, disableSlot: true }}
-              extraButtons={<ExcludeButton id={newId} />}
             />
+            {newLoc && <ExcludeButton id={newId} />}
```

Comment:
Should exclude button be conditional on `newLoc`?

Reply:
Yes because otherwise it'll continue to be rendered even after the artifact is deleted, which doesn't match the behavior of the character/set exclusion buttons (and it'd be a little weird to have an exclude button for a non-existent artifact)

Reply:
Changed `newLoc` to `newArtifact`, done